src/module.ts

```diff
@@ -231,6 +240,35 @@ export default defineNuxtModule<ModuleOptions>({
       clientSecret: '',
       redirectURL: '',
     })
+
+    // Atproto OAuth
+    for (const provider of atprotoProviders) {
+      runtimeConfig.oauth[provider] = defu(runtimeConfig.oauth[provider], atprotoProviderDefaultClientMetadata)
+    }
```

Comment:
Leaving this in the `if` block here under causes `runtimeConfig.oauth[<atprotoProvider>` not to be defined, and thus causing a type error.